### PR TITLE
New Hearst case study

### PR DESCRIPTION
**Pull request by pbakaus**
Elena/Barb, please take a final look. Thanks!

### CODE REVIEW

File: content/learn/case-studies/hearst.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      description: Increase in viewability
+    - stat: 45%
+      description: Increase in CTR on Ads
+    - stat: +1M
+      description: AMP pages published
+logo:
+  src: "case-studies/hearst_logo.png"
+  width: 60
+  height: 14
+download: "case-studies/hearst.pdf"
```
Comment (bpaduch) [inline content/learn/case-studies/hearst.md]: Where's the PDF? It's not on this PR.

File: content/learn/case-studies/hearst.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+<div class="img-right">
```
Comment (bpaduch) [inline content/learn/case-studies/hearst.md]: nit: I'm not sure if the image s/b cropped at the top but if you look at the rendered page, the image doesn't look aligned with the section it's in and the image encroaches into the next section. It's not a major issue.

File: content/learn/case-studies/hearst.md (hunk 1 above)
Comment (pbakaus) [inline content/learn/case-studies/hearst.md]: Good catch, fixed!

File: content/learn/case-studies/hearst.md (hunk 2 above)
Comment (pbakaus) [inline content/learn/case-studies/hearst.md]: mhh, looks fine to me!

**Pull request merged**